src/Microsoft.Health.Dicom.Client/Models/ChangeFeedEntry.cs

```diff
@@ -36,10 +37,13 @@ public class ChangeFeedEntry
             Action = action;
             Timestamp = timestamp;
             State = state;
+            PartitionName = partitionName;
```

Comment:
Are there any rules we need to enforce about partition names? Like can't be white space?

Reply:
It is enforced at the service level when the user issue a request. This will be anyway part of the response and not the request.